apps/sms/test/marionette/lib/messages.js

```diff
@@ -208,27 +220,21 @@ var ConversationAccessor = require('./views/conversation/accessors');
         },
 
         performHeaderAction: function() {
-          this.Composer.header.scriptWith(function(header) {
-            var event = document.createEvent('HTMLEvents');
-            event.initEvent('action', true, true);
-            header.dispatchEvent(event);
-          });
+          this.client.switchToShadowRoot(this.Composer.header);
+          this.Composer.headerActionButton.tap();
+          this.client.switchToShadowRoot();
         },
 
         performReportHeaderAction: function() {
-          this.Report.header.scriptWith(function(header) {
-            var event = document.createEvent('HTMLEvents');
-            event.initEvent('action', true, true);
-            header.dispatchEvent(event);
-          });
+          this.client.switchToShadowRoot(this.Composer.header);
+          this.Report.headerActionButton.tap();
+          this.client.switchToShadowRoot();
         },
 
         performGroupHeaderAction: function() {
-          this.Participants.header.scriptWith(function(header) {
-            var event = document.createEvent('HTMLEvents');
-            event.initEvent('action', true, true);
-            header.dispatchEvent(event);
-          });
+          this.client.switchToShadowRoot(this.Composer.header);
```

Comment:
Hm, weird, let me know if you're not able to figure this out by yourself and I'll take a look on Monday.